selenium_tests/class_roster/page_objects/class_roster_main_page.py

```diff
@@ -0,0 +1,34 @@
+"""
+This file models the Class Roster Main (Landing) Page
+"""
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium_tests.class_roster.page_objects.class_roster_base_page_object \
+    import ClassRosterBasePageObject
+
+
+class Locators(object):
+    CLASS_ROSTER_BREADCRUMB = (By.XPATH, './/h1[contains(., "Class Roster")]')
```

Comment:
This will be true for any child pages, because it will always be present in the breadcrumbs. Currently the app has no child pages, so this is fine, but we might want to fix it to avoid subtle bugs in the future if the tool changes.